test/e2e/instrumentation/logging/stackdrvier/basic.go

```diff
@@ -162,8 +162,11 @@ var _ = instrumentation.SIGDescribe("Cluster level logging implemented by Stackd
 			}()
 
 			ginkgo.By("Waiting for events to ingest")
-			zone := framework.TestContext.CloudConfig.Zone
-			c := utils.NewLogChecker(p, utils.UntilFirstEntryFromZone(zone), utils.JustTimeout, "")
+			location := framework.TestContext.CloudConfig.Zone
+			if framework.TestContext.CloudConfig.MultiZone {
```

Comment:
Isn't multi-zone different from region? What the `Zone` value will be in regional case?

Reply:
AFAIU this parameter is true for both of this cases in test env:
https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L7093

In case of regional ZONE is just a random zone from region.